pkg/lockfile/parse_test.go

```diff
@@ -95,6 +96,7 @@ func TestParse_FindsExpectedParsers(t *testing.T) {
 		"poetry.lock",
 		"pubspec.lock",
 		"requirements.txt",
+		"gradle.lockfile",
```

Comment:
`buildscript-gradle.lockfile` should be included on this list (effectively this should have all lockfiles that are listed in the readme as being supported)

Reply:
Updated to include `buildscript-gradle.lockfile`. But had to decrement the asserted count because both `gradle.lockfile` and `buildscript-gradle.lockfile` use the same parser.

Reply:
sweet as - that test is more meant to help catch people forgetting to update this section of tests if a new parser is added, which is always a bit fuzzy (arguably I should have used a `map` against the function name and then check that each was called at least once, which I think could be possible, but not worth the effort imo)